src/Compilers/Test/Core/Assert/AssertEx.cs

```
@@ -732,25 +732,6 @@ public static void AssertLinesEqual(string expected, string actual, string messa
                 expectedValueSourceLine: expectedValueSourceLine);
         }
 
-        public static void Throws<TException>(Action action, Action<TException> checker = null)
-            where TException : Exception
-        {
-            try
-            {
-                action();
```

Observe that if this didn't throw, the test might silently pass.